src/pattern-library/components/patterns/prototype/SelectNextPage.tsx

```diff
@@ -216,11 +222,31 @@ export default function SelectNextPage() {
 
           <Library.Example title="Disabled Select">
             <Library.Demo title="Disabled Select">
-              <div className="w-[350px] mx-auto">
+              <div className="w-96 mx-auto">
                 <SelectExample disabled />
               </div>
             </Library.Demo>
           </Library.Example>
+
+          <Library.Example title="Select with long content">
+            <p>
+              <code>SelectNext</code> makes sure the content never overflows,
+              but you may need to tweak <code>buttonContent</code> or options
+              content in cases where more complex content needs to be rendered.
```

Comment:
What kind of "tweak" do you mean?

Reply:
I'll try to clarify it.

Basically, the toggle button provides some affordances in the form of `text-overflow: ellipsis` and `overflow: hidden`.

While the later is always going to apply no matter the type of children provided to `buttonContent`, the former only applies when it is "plain text" (so to speak).

If you want a more complex set of children to rearrange differently, it's up to the consumer to handle it.

For example, I have included an example in the pattern library where the children have its own `truncate` wrapper to make sure the badge with the number is always displayed and only the text is hidden.

What I was trying to convey is that every use case might be different and it's up to downstream projects to customize it if the default `overflow: hidden` is not right for them.